src/styles/_catalog.scss

```diff
@@ -1,9 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px),
+    (max-width: 768px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px),
+    (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
You don't need to add min and max width together, just sort media queries in Ascending or Descending order and add only one constraint

Reply:
fixed